tests/plugins/font/_helpers/tools.js

```diff
@@ -0,0 +1,14 @@
+/* exported assertCombo */
+
+function assertCombo( comboName, comboValue, collapsed, bot, resultHtml, callback ) {
```

Comment:
If test helper has more than 3 parameters, then we tend to use `options` parameter. Additionally it would be nice to add some JSDoc for parameters.